src/chrome/content/rules/GetPocket.com.xml

```diff
@@ -1,4 +1,8 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://help.getpocket.com/favicon.ico => https://d3jyn100am7dxp.cloudfront.net/favicon.ico: 'NoneType' object has no attribute 'xpath'
+
```

Comment:
This was a bug in the checker where it tried to parse the favicon and failed. Fixed the bug and reverted this change.